FordLabs/peoplemover
# Move interfaces to Types folder

## Issue
Resolves N/A

## What was done
- [x] Move interfaces to Types folder

## How to test
1. Ensure no functionality has changed and all tests pass
2. Ensure types are stored in an organized fashion
